modules/end-user-guide/partials/proc_provisioning-jetbrains-activation-code-for-offline-use.adoc

```diff
@@ -1,55 +1,67 @@
 [id="provisioning-jetbrains-activation-code-for-offline-use_{context}"]
 = Provisioning JetBrains activation code for offline use
 
-An activation code for offline use is a file with a license code that can be retrieved from the license management section of your JetBrains account for the license that is assigned to you. When you purchase a personal subscription or are assigned a commercial subscription by your organization, you receive an email prompting you to create a JetBrains account that becomes connected with the license.
-
-NOTE: When using an activation code to activate a product, you need to generate a new activation code and apply it to your product each time the subscription is renewed.
+This section describes how to provision the JetBrains activation code to {prod-short}, when using link:https://www.jetbrains.com/store/[a JetBrains IDE requesting a registration]. After each subscription renewal, you need to generate an activation code and apply it to your product.
 
 .Prerequisites
 
-* JetBrains account
-* Personal or organization subscription
+* An active link:https://account.jetbrains.com/login[JetBrains account].
+* An active link:https://www.jetbrains.com/store/comparison.html[personal or organization subscription].
+* A `base64` conversion tool.
+* An image containing the editor, see xref:building-images-for-ides-based-on-the-intellij-platform.adoc[].
+* A workspace running with the editor, see xref:using-intellij-based-ides.adoc[].
 
 .Procedure
 
-. Retrieve the activation code from the JetBrains account:
-+
-image::jetbrains/jetbrains-account.jpg[]
-+
-JetBrains provides a zip archive with two types of the activation code. Use the `<License{nbsp}ID>{nbsp}-{nbsp}for{nbsp}2018.1{nbsp}or{nbsp}later.txt` file:
-+
-image::jetbrains/activation-code.jpg[]
+. Retrieve the activation code from the JetBrains account. JetBrains provides a zip archive with two types of the activation code. Use the `<License{nbsp}ID>{nbsp}-{nbsp}for{nbsp}2018.1{nbsp}or{nbsp}later.txt` file. Save the content in the `jetbrains_license.txt` file.
 
-. Provision the activation code for offline use with {prod-short}. This procedure is performed through the {orch-name} Secrets.
+. To define __<base64_encoded_activation_code>__, calculate the base64 encoded version of the content of the activation code.
```

Comment:
Is there a simpler way to describe this step than to`calculate the ...`?
For example,
```suggestion
. Run the `base64` command on `jetbrains_license.txt` to convert the activation code to base64 encoding.
```